components/list/demo/list-item-sample.js

```diff
@@ -0,0 +1,47 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { ListItemCheckboxMixin } from '../list-item-checkbox-mixin.js';
+//import { ListItemDragMixin } from '../list-item-drag-mixin.js';
+
+class ListItemSample extends ListItemCheckboxMixin(LitElement) {
+	// TODO: Role and breakpoints to live elsewhere
+	static get properties() {
+		return {
+			href: { type: String }
+		};
+	}
+
+	// TODO: Placeholder grid styles to be replaced with list-item-generic
+	static get styles() {
+		return [ (super.styles ? super.styles : css``), css`
+			.list-grid {
+				display: grid;
+				grid-template-columns:
+					[start outside-control-start] 40px
+					[control-start outside-control-end] 40px
+					[control-end content-start] auto
+					[content-end actions-start] auto
+					[end actions-end];
+			}
+			.d2l-list-item-content {
+				grid-column: content-start / content-end;
+				grid-row: 1 / 2;
+			}
+			.d2l-list-item-actions {
+				grid-column: actions-start / actions-end;
+				grid-row: 1 / 2;
+			}
+		`];
+	}
+	render() {
+		return html`
+			<div class="list-grid">
```

Comment:
Nit: normally I would say this should be `d2l-list-item-grid`.  For production components, we'd definitely need to be careful about this in cases where shadowDOM is not supported.

Reply:
Absolutely. In fact, this was named this way intentionally because the next PR will remove it.